packages/compose/src/hooks/use-copy-to-clipboard/index.js

```diff
@@ -13,8 +13,15 @@ import { useRef } from '@wordpress/element';
  */
 import useRefEffect from '../use-ref-effect';
 
-/** @typedef {import('@wordpress/element').RefObject} RefObject */
+/**
+ * @template T
+ * @typedef {import('react').RefObject<T>} RefObject */
```

Comment:
Is this type-def'd `RefObject` ever used?

Reply:
Whoops no that's left over.